### DISCUSSION TITLE
MK4 style load cell?

### DISCUSSION THREAD

**Discussion by Trecoolius** [lhndo/LH-Stinger#19] [Ideas]
I just ran into this printer and I love it so much that I scrapped my plans to build a voron and will build a Stinger instead. However, I'm spoiled with the MK4 load cell and I don't trust using endstops for bed leveling.

That said, are there any plans to bring modern bed leveling into this printer?

**Comment by lhndo** [comment]
Hey @Trecoolius, great to hear that!
The Stinger probing solution is provided by the Annex Engineering QuickDraw probe which is based on the tried and tested Omron D2F-5. The dock features a 0 Y offset and the printer is configured for Auto Z Tilt Calibration during homing.
